x-pack/plugins/maps/server/register_integrations.ts

```diff
@@ -35,4 +35,49 @@ export function registerIntegrations(
     shipper: 'other',
     isBeta: false,
   });
+  customIntegrations.registerCustomIntegration({
+    id: 'ingest_geojson',
+    title: i18n.translate('xpack.maps.registerIntegrations.geojson.integrationTitle', {
+      defaultMessage: 'GeoJson',
+    }),
+    description: i18n.translate('xpack.maps.registerIntegrations.geojson.integrationDescription', {
+      defaultMessage: 'Upload GeoJson with Elastic Maps.',
+    }),
+    uiInternalPath: core.http.basePath.prepend(
+      `${getFullPath('?' + OPEN_LAYER_WIZARD + '=' + WIZARD_ID.GEO_JSON)}`
+    ),
+    icons: [
+      {
+        type: 'eui',
+        src: 'logoMaps',
+      },
+    ],
+    categories: ['upload_file', 'geo'],
+    shipper: 'other',
+    isBeta: false,
+  });
+  customIntegrations.registerCustomIntegration({
+    id: 'ingest_shape',
+    title: i18n.translate('xpack.maps.registerIntegrations.shapefile.integrationTitle', {
+      defaultMessage: 'Shapefile',
+    }),
+    description: i18n.translate(
+      'xpack.maps.registerIntegrations.shapefile.integrationDescription',
+      {
+        defaultMessage: 'Upload Shapefile with Elastic Maps.',
+      }
+    ),
+    uiInternalPath: core.http.basePath.prepend(
+      `${getFullPath('?' + OPEN_LAYER_WIZARD + '=' + WIZARD_ID.GEO_JSON)}`
```

Comment:
Instead of `${getFullPath('?' + OPEN_LAYER_WIZARD + '=' + WIZARD_ID.GEO_JSON)}`, use 
```
getFullPath(`?${OPEN_LAYER_WIZARD}=${WIZARD_ID.GEO_JSON}`)
```

Also, make this a constant in this file and use in both calls to registerCustomIntegration